src/System.Reflection.Metadata/src/System/Reflection/BlobBuilder.Enumerators.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Diagnostics;
+using System.Collections;
+using System.Collections.Generic;
+using System;
+
+#if SRM
+namespace System.Reflection
+#else
+namespace Roslyn.Reflection
+#endif
+{
+    partial class BlobBuilder
+    {
+        // internal for testing
```

Comment:
Yeah, I'd keep them in Roslyn for now to make it easier to iterate (I need to keep these sources in sync). Be assured that the entire Roslyn test suite is testing this code.